content/rest/usage.textile

```diff
@@ -106,9 +106,21 @@ blang[php].
   ```
 
 blang[java].
-  The REST library for Java and Android is "hosted on Github":https://github.com/ably/ably-java and is downloadable as a JAR from "https://github.com/ably/ably-java/releases":https://github.com/ably/ably-java/releases or via "JCentre":https://bintray.com/ably-io/ably.
+  The REST library for Java and Android is "hosted on Github":https://github.com/ably/ably-java and can be used by adding one line to @build.gradle@ dependencies section.
 
-  Ensure the JAR is included in your classpath as follows:
+  For Java applications:
+
+  ```[java]
+  compile 'io.ably:ably-java:0.8.7'
+  ```
+
+  For Android apps:
+
+  ```[java]
+  compile 'io.ably:ably-android:0.8.7'
+  ```
+
+  Ensure the library is included in your classpath as follows:
```

Comment:
Did we not agree we'd mention that the version number should be updated with perhaps links to the latest version numbers?